programs/vest_api/src/vest_state.rs

```diff
@@ -0,0 +1,110 @@
+//! vest state
+use crate::vest_schedule::create_vesting_schedule;
+use bincode::{self, deserialize, serialize_into};
+use chrono::prelude::*;
+use chrono::{
+    prelude::{DateTime, TimeZone, Utc},
+    serde::ts_seconds,
+};
+use serde_derive::{Deserialize, Serialize};
+use solana_sdk::{account::Account, instruction::InstructionError, pubkey::Pubkey};
+
+#[derive(Serialize, Deserialize, Debug, Clone, PartialEq)]
+pub struct VestState {
+    /// The address authorized to terminate this contract with a signed Terminate instruction
+    pub terminator_pubkey: Pubkey,
+
+    /// The address authorized to redeem vested tokens
+    pub payee_pubkey: Pubkey,
+
+    /// The day from which the vesting contract begins
+    #[serde(with = "ts_seconds")]
+    pub start_dt: DateTime<Utc>,
+
+    /// Address of an account containing a trusted date, used to drive the vesting schedule
+    pub date_pubkey: Pubkey,
+
+    /// The number of lamports to send the payee if the schedule completes
+    pub lamports: u64,
```

Comment:
What do you think about renaming this field to make it more distinct from the actual lamport balance of the contract account? There are a lot of `lamport` variables being credited and debited in the redeem_tokens method below, and it might be useful to distinguish this one to help indicate it won't ever change after the initialization of the account.

Reply:
Now `total_lamports`